scripts/build_gas_network.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Builds clustered natural gas network based on data from:
+    [1] the SciGRID Gas project
+        (https://www.gas.scigrid.de/)
+    [2] ENTSOG capacity map
+        (https://www.entsog.eu/sites/default/files/2019-10/Capacities%20for%20Transmission%20Capacity%20Map%20RTS008_NS%20-%20DWH_final.xlsx)
+
+Relevant Settings
+-----------------
+.. code:: yaml
+    sector:
+
+
+Inputs
+------
+gas network data from SciGRID gas and ENTSOG:
+    - gas_network="data/gas_network/gas_network_dataset.csv",
+      combined gas network data set from [1] and [2]
+    - country_shapes=pypsaeur("resources/country_shapes.geojson"),
+    - regions_onshore=pypsaeur("resources/regions_onshore_elec_s{simpl}_{clusters}.geojson"),
+    - regions_offshore=pypsaeur("resources/regions_offshore_elec_s{simpl}_{clusters}.geojson")
+
+Outputs
+-------
+clustered gas network data for corresponding PyPSA-Eur-Sec network
+    - gas_network='resources/gas_network_{clusters}.csv'
+
+Description
+-----------
+
+"""
+import geoplot
+import geoplot.crs as gcrs
```

Comment:
- [ ] needs to be added to an `environment.yaml` (currently in PyPSA-Eur)

Reply:
I think the geoplot package is only used for the [visualisation functions] (https://github.com/PyPSA/pypsa-eur-sec/blob/81b009e0623f149c2c7695bd83ac56e43d5ae9fd/scripts/build_gas_network.py#L226). We can remove them and the package as well

Reply:
We could move them to a Github gist and reference it at the top of the file so it's not completely lost.

Reply:
https://gist.github.com/fneum/0fff6bf2e45eb75496eea1047278cc1b